learningresources/api.py

```diff
@@ -41,19 +41,21 @@ class NotFound(LearningResourceException):
     pass
 
 
+# pylint: too-many-arguments
```

Comment:
i guess you are missing the `disable=` here, but probably if you put at this level it will disable it for all the functions